src/pages/home/ReportScreen.js

```diff
@@ -293,6 +296,7 @@ class ReportScreen extends React.Component {
                                         isComposerFullSize={this.props.isComposerFullSize}
                                         isDrawerOpen={this.props.isDrawerOpen}
                                         parentViewHeight={this.state.skeletonViewContainerHeight}
+                                        isChatReportEmptyWhileOffline={isChatReportEmptyWhileOffline}
```

Comment:
Continuing the previous thought.. we are creating this new variable and prop - but nothing else uses `isChatReportEmptyWhileOffline` so it seems like the responsibility of the `ReportActionsView` to:

1. Check the `reportActions`
2. See if we're offline
3. If there are no `reportActions` and we are offline then incorporate the fallback action